viz/charts/linkerd-viz/templates/tap-injector-rbac.yaml

```diff
@@ -0,0 +1,95 @@
+###
+### Tap Injector RBAC
+###
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{.Values.namespace}}-tap-injector
```

Comment:
Given this is a cluster-level resource, and to follow the same scheme as the Jaeger injector, I don't think it's necessary to parametrize by the namespace, and instead hard-code as `linkerd-tap-injector` so it's easily identified (same comment for the resources below).